app/helpers/table_helper.rb

```diff
@@ -0,0 +1,61 @@
+module TableHelper
+  def sort_table_header(heading, attribute_name)
+    SortTable.new(self, heading, attribute_name).render
+  end
+
+  class SortTable
+    attr_accessor :view, :heading, :attribute_name
+
+    delegate :content_tag, :params, :link_to, :organisation_content_items_path, to: :view
+
+    def initialize(view, heading, attribute_name)
+      @view = view
+      @heading = heading
+      @attribute_name = attribute_name
+    end
+
+    def render
+      content_tag :th, "aria-sort" => aria_label do
+        link text_label, order_param
+      end
+    end
+
+  private
+
+    def link(label, order)
+      link_to organisation_content_items_path(organisation_id: view.instance_variable_get(:@organisation), sort: attribute_name, order: order) do
+        "#{heading} #{content_tag :span, label, class: 'visually-hidden'}".html_safe
+      end
+    end
+
+    def currently_sorted_by?
+      params[:order].present? && params[:sort] == attribute_name
+    end
+
+    def text_label
+      param = order_param
+      "sort by #{heading}, #{readable_sort_term(param)}".downcase
```

Comment:
What's hidden in the `th` would be read out by screen readers in every `td` underneath. Therefore I would try to keep this as short as possible and not duplicate anything. I would suggest something like "sort column descending" or just "sort descending".